Snakefile

```diff
@@ -121,8 +121,20 @@ def get_outputs():
     if 'atlas-experiment-summary' in config['tool'] or 'all' in config['tool']:
         outputs.append(f"{config['accession']}-atlasExperimentSummary.Rdata")
     if 'baseline-heatmap' in config['tool'] or 'all-baseline' in config['tool']:
-        outputs.extend(expand(f"{config['accession']}"+"-{metric}.tsv", metric=get_metrics() ))
+        outputs.extend(expand(f"{config['accession']}"+"-heatmap-{metric}.pdf", metric=get_metrics() ))
+    if 'baseline-coexpression' in config['tool'] or 'all-baseline' in config['tool']:   
+        outputs.extend(expand(f"{config['accession']}"+"-{metric}-coexpressions.tsv.gz", metric=get_metrics() )) 
     print(outputs)
+    print('Getting list of outputs.. done')
+    
+    if config['delete_previous_output']==True:
```

Comment:
```suggestion
    if 'delete_previous_output' in config and config['delete_previous_output']==True:
```
otherwise it might complain if the flag is not present